setup/scripts/setup/setup_subgraph.sh

```diff
@@ -0,0 +1,25 @@
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+
+# Handle node version
+node_version=`node -v`
+export NVM_DIR=$HOME/.nvm;
+source $NVM_DIR/nvm.sh;
+
+if [ "$node_version" != "v16.15.1" ]; then
+  echo $node_version
+  nvm install 16.15.1
+fi
+
+nvm use
+npm --prefix packages/subgraph run start-local
+# wait for subgraph to start/initialize
+sleep 45 
```

Comment:
kinda hacky, if it takes longer then 45 seconds will fail, but I guess its ok...